src/Raven.Server/Documents/Includes/IncludeDocumentsDuringTimeSeriesLoadingCommand .cs

```diff
@@ -17,6 +18,7 @@ public unsafe class IncludeDocumentsDuringTimeSeriesLoadingCommand
         private byte* _state;
         private readonly Dictionary<string, BlittableJsonReaderObject> _includesDictionary;
         private DynamicJsonValue _includes;
+        private HashSet<string> _missingIncludes;
```

Comment:
Do we need this hashset to be case-insensitive?

Reply:
let's have a test with including `users/1` & `Users/1`

Reply:
changed and added test